package/abs/Dockerfile

```diff
@@ -0,0 +1,27 @@
+############################################################
+# Dockerfile to build Rancher Aliyun Block Storage Containers
+# Based on Ubuntu
+############################################################
+
+# Set the base image to Ubuntu 16.04
+FROM ubuntu:16.04
+
+# Change apt source
+# COPY abs/aliyun.list /etc/apt/sources.list.d/
```

Comment:
I forget to delete it. This file can help me to improve the speed when updating Ubuntu at home.